src/LibFLExBridge-ChorusPluginTests/Handling/DateTimeMergingTests.cs

```diff
@@ -1,242 +0,0 @@
-﻿// --------------------------------------------------------------------------------------------
```

Comment:
I'm sure these tests just moved, but I can't tell.

Reply:
Nope, the merge went wrong. Got deleted in the merge. Thanks for noticing this.